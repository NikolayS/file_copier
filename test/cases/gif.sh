#!/bin/bash

# to run this scenario, COPIERHOST should be defined (`export COPIERHOST=http://your.copier.host.name`)

http GET "http://$COPIERHOST/?wh=1&md5=1&src=https://media.giphy.com/media/YU92Dp0cTqz3q/giphy.gif" -h | grep -v "Date:"

