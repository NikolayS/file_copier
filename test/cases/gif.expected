HTTP/1.1 200 OK
Server: nginx
Transfer-Encoding: chunked
X-File-Copier-Md5: 379d7705580ec151d17a466f16fcbb08
X-File-Copier-Img-Width: 460
X-File-Copier-Img-Height: 345
X-File-Copier-Size: 601086
X-Location: /data/00/89/9e/4f/00899e4f3d177b58b90957e8d25215cf6ede0ab49bb66099eaa96ab6f43df54e.gif
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5
DO_FAIL_PLEASE
